src/transformStateWithClones.js

```diff
@@ -6,8 +6,36 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newArr = [];
+
+  for (const item in actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
Це масив. Тут потрібен цикл for of